### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by eltrubetskaya**

### CODE REVIEW

File: src/AppBundle/Entity/Role.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ class Role extends AbstractPersonalTranslatable implements TranslatableInterface
      * @var Performance
      *
      * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Performance", inversedBy="roles")
+     * @Type("array")
```
Comment (spolischook) [inline src/AppBundle/Entity/Role.php]: Why? It contains entity

File: src/AppBundle/Entity/Role.php [hunk 2]
```diff
@@ -65,7 +66,7 @@ class Role extends AbstractPersonalTranslatable implements TranslatableInterface
      * @var Employee
      *
      * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Employee", inversedBy="roles")
-     * @Type("AppBundle\Entity\Employee")
+     * @Type("array")
```
Comment (spolischook) [inline src/AppBundle/Entity/Role.php]: Why? It contains entity

Comment (eltrubetskaya) [inline src/AppBundle/Entity/Role.php]: this return object in response, but in doc must be array. Swagger message in error
[https://github.com/geekhub-php/CheTheatre/blob/master/doc/theatre.json#L177](url)
change doc?

Comment (spolischook) [inline src/AppBundle/Entity/Role.php]: Yes, it should be an object. And check the validator - it should get an error about that.

Comment (eltrubetskaya) [inline src/AppBundle/Entity/Role.php]: Validator issued an this error

Comment (eltrubetskaya) [inline src/AppBundle/Entity/Role.php]: https://github.com/geekhub-php/CheTheatre/blob/master/doc/theatre.json#L480 
this must return object in response too?

Comment (spolischook) [inline src/AppBundle/Entity/Role.php]: This is ok. There are array of objects

Comment (eltrubetskaya) [inline src/AppBundle/Entity/Role.php]: ok

Comment (spolischook) [inline src/AppBundle/Entity/Role.php]: @eltrubetskaya API resource MUST return either Object[] or Object in body. This rule should be applied to any resource within REST API

**Pull request merged**